modules/virt-creating-data-volumes-using-pvc-api.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virtual_disks/virt-creating-data-volumes.adoc
+
+[id="virt-creating-data-volumes-using-pvc-api_{context}"]
+
+= Creating data volumes using the PVC API
+
+When you create a data volume using the *pvc* API, the CDI creates the data volume based on your explicit:
+
+* `accessModes` (`ReadWriteOnce`, `ReadWrtieMany`, `ReadOnlyMany`)
+* `volumeMode` (`Filesystem` or `Block`)
+* `capacity` of `storage` (`5Gi`, for example)
+
+When you explicitly allocate a data volume by using the *pvc* API, you must consider filesystem overhead. Filesystem overhead is the amount of space required by the CDI to manage the file. Failing to account for filesystem overhead in your storage capacity request may result in an underlying PVC that is not large enough to accommodate your virtual machine disk.
+
+If you use the *storage* API, the CDI will factor in filesystem overhead and  request a larger PVC to ensure that your allocation request is successful.
+
+In the following YAML, using the *pvc* API allocates a data volume with a storage capacity of two gigabytes. In this case, you specify an access mode of `ReadWriteMany` to ensure that many nodes can mount the volume for reading and writing. Because you know the values your system can support, you specify `Block` storage instead of `Filesystem`, using the `volumeMode` attribute.
+
+[source,yaml]
+----
+apiVersion: cdi.kubevirt.io/v1beta1
+kind: DataVolume
+metadata:
+  name: <datavolume> <1>
+spec:
+  source:
+    pvc:
```

Comment:
Should we explain (with a number) that this pvc: indicates that the source of the import is another existing pvc (ie a clone)?

Reply:
Yes